code/index.html

```diff
@@ -1,30 +1,106 @@
+            </picture>
+        </header>
+
+        <div id="menu_button" class="menu">
+            <p class="menutxt">MENU</p>
+        </div>
+        <ul>
+            <li>
+                <a href="#">
+                    About</a>
+            </li>
+            <li>
+                <a href="#">
+                    FAQ</a>
+            </li>
+            <li>
+                <a href="form.html">
+                    FORM</a>
+            </li>
+        </ul>
+
+        <div class="main">
+            <div id="questions">
+                <h1 id="cookiequestions">Cookie Questions</h1>
+                <button class="dropacc">Do you have fat free cookies</button>
+                <div class="txt">
+                    <p>Yes we do, Lorem, ipsum dolor sit amet consectetur adipisicing elit. Officia
+                        atque suscipit ea voluptatem repellendus a ipsam consectetur culpa ullam?
+                        Magnam.Lorem ipsum dolor sit amet, consectetur adipisicing elit. Molestiae
+                        atque, repellendus sequi dignissimos optio culpa!</p>
+                </div>
+
+                <button class="dropacc">Do you have gluten free cookies</button>
+                <div class="txt">
+                    <p>Yes we do, Lorem Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                        Iure soluta molestias est reiciendis vel quasi deleniti, alias enim iste nulla
+                        totam illum placeat. Cupiditate laudantium cum nisi amet quos qui? ipsum dolor
+                        sit amet, consectetur adipisicing elit. Molestiae atque, repellendus sequi
+                        dignissimos optio culpa!</p>
+                </div>
+
+                <button class="dropacc">How many cookies can i order?</button>
+                <div class="txt">
+                    <p>Subscribe to our unlimited cookie delivery, Lorem ipsum dolor sit amet,
+                        consectetur adipisicing elit. Molestiae atque, repellendus sequi dignissimos
+                        optio culpa!</p>
+                </div>
+
+                <button class="dropacc">My cookies crumbled</button>
+                <div class="txt">
+                    <p>Impossible, Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia,
+                        necessitatibus inventore! Dignissimos excepturi odio aliquam quidem nostrum
+                        repellat sunt eos, alias deserunt excepturi fugiat.dolor sit amet, consectetur
+                        adipisicing elit. Molestiae atque, repellendus sequi dignissimos optio culpa!</p>
+                </div>
+            </div>
+            <div class="linktoform">
+                <a href="form.html">
+                    <p>Klick here to fill the form!</p>
+                </a>
+            </div>
+        </div>
+
+        <footer id="footer">
+            <p>Stay in touch
+            </p>
+            <a href="lisaojeland@gmail.com">✉️</a>
```

Comment:
Tip! You can make this link to be clickable with the mailto addition. Then if someone will click it their mail software will open up which is kind of nice
```suggestion
            <a href="mailto:lisaojeland@gmail.com">✉️</a>
```